go/oasis-node/cmd/node/node_control.go

```diff
@@ -332,7 +332,7 @@ func (n *Node) getRuntimeStatus(ctx context.Context) (map[common.Namespace]contr
 		}
 
 		// Fetch provisioner type.
-		_, provisioner, err := rt.Host(ctx)
+		_, provisioner, err := rt.Host()
 		switch {
 		case err != nil:
```

Comment:
Should we also set provisioner to something if there is an error? Maybe `"unknown"` or `"pending"` similar?

Reply:
`"pending"` makes sense to me